build-support/bin/ci.py

```diff
@@ -458,7 +458,8 @@ def run_rust_tests() -> None:
   command = [
     "build-support/bin/native/cargo",
     "test",
-    "--all",
+    # Until we can run fuse tests in osx CI, we omit --all.
+    #"--all",
```

Comment:
(I know you're still experimenting at this stage)

Would feel better if we could parameterize this function so that Linux shard still uses `--all`.

Reply:
Done.